packages/web_benchmarks/lib/src/runner.dart

```diff
@@ -55,6 +55,7 @@ class BenchmarkServer {
     required this.chromeDebugPort,
     required this.headless,
     required this.treeShakeIcons,
+    this.useWasm = false,
```

Comment:
I wonder if this will have the same effect as `useCanvasKit` above. `useCanvasKit` ended up being not extensible enough, so we'll need to make a breaking change to accommodate Skwasm. I wonder if we should instead have a `CompilationOptions` object that can be extended in the future. cc @eyebrowsoffire

Reply:
I added a CompilationOptions class. PTAL @yjbanov @eyebrowsoffire